cypress/integration/apps/reportes-diarios/reportes-diarios.spec.js

```diff
@@ -0,0 +1,147 @@
+            fechaPrestacion:  moment().startOf('months').subtract(1, 'day').add(ahora.hour(), 'hour'),                          // Ultimo dia del mes
+        },
+    ]
+    // const inicioMesAnterior = moment().startOf('months').subtract(1, 'months').add(ahora.hour(), 'hour');
+    // const inicioMesActual = moment().startOf('months').subtract(ahora.hour(), 'hour');;
+    // const mes = inicioMesActual - inicioMesAnterior;
+
+    before(() => {
+        cy.seed();
+
+        // const rangoEtarios = [
+        //     15,             // Menos de un Año - dias
+        //     180,            // Menos de un Año - Meses
+        //     365.25 * 2,     // 1 a 4
+        //     365.25 * 9,     // 5 a 14
+        //     365.25 * 17,    // 15 a 19
+        //     365.25 * 29,    // 20 a 39
+        //     365.25 * 54,    // 40 a 69
+        //     365.25 * 80,    // 70 y mas
+        // ];
+
+
+        for(const param of params) {
+            const diasPrestacion = moment.duration(param.fechaPrestacion - ahora).asDays();
+            cy.task('database:create:paciente', {sexo: 'masculino', fechaNacimiento: param.fechaNacimiento })
+                .then(p => cy.task('database:seed:prestacion', { 
+                    paciente: p._id,
+                    estado: 'validada',
+                    fecha: diasPrestacion
+                }));
+            
+            cy.task('database:create:paciente', {sexo: 'femenino', fechaNacimiento: param.fechaNacimiento })
+                .then(p => cy.task('database:seed:prestacion', { 
+                    paciente: p._id,
+                    estado: 'validada',
+                    fecha: diasPrestacion
+                }));
+        }
+
+        mesIndex = params[0].fechaPrestacion.get('month');
+
+        cy.login('30643636', 'asd')
+            .then(t => token = t);
+    })
+
+    beforeEach(() => {
+        cy.server();
+        cy.route('GET', '**/api/core/tm/organizaciones**').as('organizaciones');
+        cy.route('GET', '**/api/core/tm/tiposPrestaciones**').as('prestaciones');
+        cy.goto('/reportesDiarios', token);
+    });
+
+    it('Generar reportes diarios', () => {        
+        cy.plexSelectAsync('label="Organización"', 'HOSPITAL PROVINCIAL NEUQUEN', '@organizaciones', 0);
+        cy.plexSelect('label="Seleccione reporte"', 0)
+            .click();
+        cy.plexSelect('label="Mes"', mesIndex)
+            .click();
+        cy.plexSelect('label="Año"', ultimoAnioIndex - (ahora.year() - params[0].fechaPrestacion.year()))
+            .click();
+        cy.plexSelectAsync('label="Prestación"', 'consulta con medico general', '@prestaciones', 0);
+
+        cy.plexButton('Generar Reporte').click();
+        for (let i=0; i<params.length; i++) {
+            cy.get('table>tbody>tr').eq(params[i].fechaPrestacion.date() - 1).within(() => {
+                // Validacion de rango etario
+                cy.get('td').eq(i * 2 + 1).contains(1);
```

Comment:
Este tipo de chequeo debería hacerse con la propiedad should: `cy.get('td').eq(i * 2 + 1).should('contain', 1);` 

ademas de esta forma podemos visualizar el acierto/fallo
![test-reporte](https://user-images.githubusercontent.com/37875706/80634837-6faf2000-8a31-11ea-9afd-24cfcedeb1f9.png)

Lo demas bien!

Reply:
Listo, arreglado!